### ISSUE TITLE
Remove OpenSSL from `alpine-slim` images

### ISSUE THREAD

**Issue by ferferga** [nginx/docker-nginx#942]
### Is your feature request related to a problem? Please describe

I use this image for shipping a frontend application that just needs file serving. SSL handling is done with another reverse proxy, so the extra weight (4 MB+ uncompressed, which is quite noticeable) by OpenSSL is wasteful and could be improved

### Describe the solution you'd like

As stated in the docs, the alpine slim images are intended for the most basic cases (like mine). For those tags, nginx should be built without SSL support, leaving the normal images for users who desire that.

### Describe alternatives you've considered

* Remove openssl using apk
* Remove the linked libraries manually in a build step, but nginx has a hard requirement on them.
* Patching the binary with `patchelf`, but it doesn't start even when forcing ``ssl off`` in the main http block.

**Comment by thresheek** [comment]
Agreed, SSL support is something we will not remove from the images.

**Comment by thresheek** [comment]
Closing as wontfix.
